KIPs/kip-11.md

```diff
@@ -0,0 +1,126 @@
+
+Through this yield farming process and the marketing activities involved, it is also more likely that power users will make the effort to understand Kyber Network and the benefits of its various protocols such as KyberDMM. 
+
+In addition, the tokens of popular projects often generate **considerable trading volume** compared to others and this directly benefits KNC holders as 10% of KyberDMM volume is given to KNC holders who vote on KyberDAO (the other 90% goes to liquidity providers). Higher trading volume will increase the fees earned by the protocol and liquidity providers (APY), as well as the value of KNC. This helps ensure the sustainability of the KyberDMM protocol.
+
+
+## Framework: Joint Liquidity Mining Proposal
+
+Projects can submit proposals to get their tokens voted into KyberDMM’s Rainmaker program to receive liquidity mining rewards on top of protocol fees. KNC rewards received during liquidity mining can be restaked into the KNC/ETH pool for even more rewards.
+
+Although proposal details may differ depending on the needs of each individual project, all tokens have to fulfil certain criteria based on this liquidity mining framework in order to be eligible for a KyberDAO vote and Rainmaker liquidity mining.
+
+Proposals have to be first submitted to Kyber’s governance [forum](https://gov.kyber.org/c/collaborations) for discussion for 5-7 days, where the Kyber community will discuss and determine the eligibility of a project and its KNC reward allocation.
+
+Projects have to submit their proposal on this [Github form](https://github.com/p-thudo/KIPs/issues/new?assignees=&labels=proposal%2Ctriage&template=joint_mining_proposal.yml&title=%5BProposal+from%5D%3A+) as well as the Kyber governance [forum](https://gov.kyber.org/c/collaborations). It is recommended that proposals follow the template given below.
+
+### The template
+
+1. Project Introduction
+2. Proposal Summary, Motivation and Key Details (refer to the selection criteria guide)
+3. KNC Amount Requested
+4. Amount of Tokens Contributed by your Project
+5. Network: Ethereum or Polygon
+6. Vesting: No vesting or 14 days vesting
+7. Campaign start date and duration: 1-3 months
+8. Project details: Contact information, website url, logo, token contract address, link to CoinGecko/Coinmarketcap markets page etc.
+
+### Example
+
+**1. Project Introduction**
+XYZ Finance is a DeFi project that...
+
+**2. Proposal Summary, Motivation and Key Details**
+*Refer to the selection criteria guide below.*
+
+XYZ Finance is proposing a joint liquidity mining program with Kyber to help improve liquidity for the XYZ token on the capital efficient KyberDMM protocol.
+
+During the campaign period, liquidity providers who add liquidity (deposit tokens) on the eligible XYZ-ETH pool will receive LP tokens that can be staked on the Yield page for additional XYZ and KNC token rewards. As part of the campaign launch, there will also be joint marketing efforts with Kyber Network.
```

Comment:
is it the pool fixed? like just XYZ-ETH pool?

Reply:
Partners should have options on which token they want to pair with in the rewarded pool. This also should be asked in the Joint LM proposal form. Please consult on list of token option @tranvictor.

Reply:
ETH, WBTC, USDT, USDC, DAI and KNC are the options we can propose.